auth/providers/azure/graph/graph_test.go

```diff
@@ -122,9 +122,12 @@ func TestLogin(t *testing.T) {
 		if !time.Now().Before(u.expires) {
 			t.Errorf("Expiry not set properly. Expected it to be after the current time. Actual: %v", u.expires)
 		}
+		// Normalize to second precision for comparison
+		expectedExpiresOn := expiresOn.Add(-tokenExpiryDelta).Truncate(time.Second)
+		actualExpires := u.expires.Truncate(time.Second)
 		// Expect u.expires to be tokenExpiryDelta before expiresOn
```

Comment:
this comment is not relevant anymore? or might be confusing given that you're normalizing it above for comparision